### PR TITLE
qgis: 3.26.0 → 3.26.1, qgis-ltr: 3.22.8 → 3.22.9

### PR DESCRIPTION
**Pull request by sikmir**
###### Description of changes
Update qgis and qgis-ltr to today's releases.

###### Things done
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by willcohen — APPROVED** [review]

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -25526,6 +25519,38 @@ with pkgs;
 
   zuki-themes = callPackage ../data/themes/zuki { };
 
+  ### APPLICATIONS / GIS
```
Comment (erictapen) [inline pkgs/top-level/all-packages.nix:25522]: I'm not sure about this grouping, do we do it on any other place? The current structure of all-packages is messy, but I think there should be some kind of standardisation process before actively changing the grouping from top-level categories to something else.

Comment (sikmir) [reply pkgs/top-level/all-packages.nix:25522]: Another example: https://github.com/NixOS/nixpkgs/pull/178376/commits/d35fc529cf09512aa28f0d78489e122350f20c53
I think it's inline with comment:
```
/* The top-level package collection of nixpkgs.
 * It is sorted by categories corresponding to the folder names in the /pkgs
 * folder. Inside the categories packages are roughly sorted by alphabet, but
 * strict sorting has been long lost due to merges. Please use the full-text
 * search of your editor. ;)
 * Hint: ### starts category names.
 */
```
Just a small step to improve structure.

Comment (erictapen) [reply pkgs/top-level/all-packages.nix:25522]: Just skimmed through the file and your pattern seems to be more common than I expected. Thanks for the explanation.

**Comment by erictapen** [comment]
I merge, even though aarch64-darwin didn't pass yet, as it didn't run through in 3 days.

**Comment by sikmir** [comment]
> even though aarch64-darwin didn't pass yet

Darwin support for qgis is not finished yet (#157862)

**Pull request merged**